pegasus/sites.v3/code.org/public/private/poste/index.haml

```diff
@@ -83,7 +83,8 @@
     Showing most recently sent mail. Click to view all mail.
 
 :ruby
-  rows = DB.fetch("
+  POSTE_REPORTING_DB_READONLY = sequel_connect(CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader)
```

Comment:
Hmm, I didn't want to include a file for a single definition. Is `lib/cdo/db.rb` already auto-included in this file? If so, I'll just use the existing `PEGASUS_REPORTING_DB_READONLY`.